packages/angular_devkit/build_angular/src/webpack/configs/typescript.ts

```diff
@@ -51,20 +51,20 @@ function createIvyPlugin(
     }
   }
 
-  let inlineStyleMimeType;
+  let inlineStyleFileExtension;
   switch (buildOptions.inlineStyleLanguage) {
     case 'less':
-      inlineStyleMimeType = 'text/x-less';
+      inlineStyleFileExtension = 'less';
       break;
     case 'sass':
-      inlineStyleMimeType = 'text/x-sass';
+      inlineStyleFileExtension = 'sass';
       break;
     case 'scss':
-      inlineStyleMimeType = 'text/x-scss';
+      inlineStyleFileExtension = 'scss';
       break;
     case 'css':
     default:
-      inlineStyleMimeType = 'text/css';
+      inlineStyleFileExtension = 'css';
       break;
   }
```

Comment:
Less verbose:
```ts
if (['less', 'sass', 'scss', 'css'].includes(buildOptions.inlineStyleLanguage)) {
  inlineStyleFileExtension = buildOptions.inlineStyleLanguage;
}
```

Or if `buildOptions.inlineStyleLanguage` is already validated based on the schema then you can simplify even more